plugins/rbac-backend/src/service/router.ts

```diff
@@ -2,10 +2,10 @@ import { errorHandler } from '@backstage/backend-common';
 
 import express from 'express';
 import Router from 'express-promise-router';
-import { Logger } from 'winston';
+import { LoggerService } from '@backstage/backend-plugin-api';
 
```

Comment:
Similarly for the rbac-backend, the `loggerToWinstonLogger` wrappers should be removed:
https://github.com/janus-idp/backstage-plugins/blob/835e63f3995fb840108118681f6e012f7ac88f08/plugins/rbac-backend/src/plugin.ts#L1